src/test/java/org/togetherjava/discordbot/util/DurationFormatterTest.java

```diff
@@ -0,0 +1,76 @@
+package org.togetherjava.discordbot.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import static org.assertj.core.api.Assertions.*;
+class DurationFormatterTest {
+
+  private DurationFormatter durationFormatterUnderTest;
+
+  @BeforeEach
```

Comment:
I see no point in creating a new `DurationFormatter` before each test, maybe use `@BeforeAll`? Also, we may want to convert `DurationFormatter` to a static util class; however this can be done at a later date.
```suggestion
  @BeforeAll
```